birdhouse/scripts/read-configs.include.sh

```diff
@@ -166,19 +177,40 @@ read_components_default_env() {
 
     # EXTRA_CONF_DIRS and DEFAULT_CONF_DIRS relative paths are relative to COMPOSE_DIR.
     if [ -d "$COMPOSE_DIR" ]; then
-        cd "$COMPOSE_DIR"
+        cd "$COMPOSE_DIR" >/dev/null
     fi
 
     source_conf_files "$DEFAULT_CONF_DIRS" 'DEFAULT_CONF_DIRS'
     source_conf_files "$EXTRA_CONF_DIRS" 'EXTRA_CONF_DIRS'
 
     # Return to previous pwd.
     if [ -d "$COMPOSE_DIR" ]; then
-        cd -
+        cd - >/dev/null
     fi
 }
 
 
+check_optional_vars() {
+
+    for i in ${OPTIONAL_VARS}
+    do
+        v="${i}"
```

Comment:
Can we change these single letter variable names to something more descriptive.

And/or add a comment to describe what this is doing.

Reply:
I simply used the same nomenclature used by similar loops.
I think their scope is sufficiently small to understand their use without being over explicit by comments, but I can adjust if that is judged really necessary.

Reply:
I don't care that much I guess